test/todo.js

```diff
@@ -15,7 +15,7 @@ tap.test('tape todo test', function (assert) {
             'TAP version 13\n'
             + '# success\n'
             + 'ok 1 this test runs\n'
-            + '# failure\n'
+            + '# TODO failure\n'
```

Comment:
hm, there’s a suffix on all the assertions, why would the description get a prefix?

Reply:
This was to maintain consistency with `# SKIP something`